security/cert_manager_operator/cert-manager-operator-monitoring.adoc

```diff
@@ -0,0 +1,21 @@
+:_content-type: ASSEMBLY
+[id="cert-manager-operator-monitoring"]
+= Enabling monitoring for the {cert-manager-operator}
```

Comment:
Please replace `the {cert-manager-operator}` with `cert-manager` so it has parity with what is in the TOC.

These two got different titles as of now.
![image](https://github.com/openshift/openshift-docs/assets/10649800/5be06498-25ff-4c84-8aad-02b7865addb9)